### ISSUE TITLE
Nose chondritis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6514]
**Preferred term label:**
Nose chondritis

**Synonyms**
Nasal chrondritis

**Definition (free text, please give PubMed ID)**
Inflammation of the cartilage of the nose.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0100662
(Also add chondritis of pinna)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
16/25
301054 
PMID:33108101

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
